### PR TITLE
update illuminate up to 6.x

### PR DESCRIPTION
**Pull request by korobovn**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | No

### Changed

- Maximal `illuminate/*` packages version now is `6.*`

### Added

- GitHub actions for a tests running

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I wrote unit tests for my code
- [x] I have made changes in [CHANGELOG.md](https://github.com/avto-dev/app-version-laravel/blob/master/CHANGELOG.md) file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,9 +21,11 @@ before_install:
   - if [[ $coverage = 'false' ]]; then phpenv config-rm xdebug.ini || true; fi
 
 install:
-  - if [[ $setup = 'basic'  ]]; then travis_retry composer update --prefer-dist --no-interaction --no-suggest; fi
+  - if [[ $setup = 'basic'  ]] && [[ $laravel = 'default'  ]]; then travis_retry composer update --prefer-dist --no-interaction --no-suggest; fi
```
Comment (tarampampam) [inline .travis.yml]: Without ` && [[ $laravel = 'default'  ]]` build fails?

File: .travis.yml [hunk 2]
```diff
@@ -20,9 +21,11 @@ before_install:
...
   - if [[ $setup = 'stable' ]]; then travis_retry composer update --prefer-dist --no-interaction --no-suggest --prefer-stable; fi
   - if [[ $setup = 'lowest' ]]; then travis_retry composer update --prefer-dist --no-interaction --no-suggest --prefer-lowest; fi
+  - if [[ $laravel != 'default' ]]; then travis_retry composer require --dev --update-with-dependencies --prefer-dist --no-interaction --no-suggest laravel/laravel "$laravel"; fi
```
Comment (tarampampam) [inline .travis.yml]: Not `laravel/laravel`. `illuminate/support`

File: .travis.yml [hunk 3]
```diff
@@ -8,6 +8,7 @@ env:
   global:
     - setup=basic
     - coverage=false
+    - laravel=default
```
Comment (tarampampam) [inline .travis.yml]: `laravel` -> `illuminate`

File: .travis.yml [hunk 4]
```diff
@@ -20,9 +21,11 @@ before_install:
   - if [[ $coverage = 'false' ]]; then phpenv config-rm xdebug.ini || true; fi
 
 install:
-  - if [[ $setup = 'basic'  ]]; then travis_retry composer update --prefer-dist --no-interaction --no-suggest; fi
+  - if [[ $setup = 'basic'  ]] && [[ $laravel = 'default'  ]]; then travis_retry composer update --prefer-dist --no-interaction --no-suggest; fi
   - if [[ $setup = 'stable' ]]; then travis_retry composer update --prefer-dist --no-interaction --no-suggest --prefer-stable; fi
   - if [[ $setup = 'lowest' ]]; then travis_retry composer update --prefer-dist --no-interaction --no-suggest --prefer-lowest; fi
+  - if [[ $laravel != 'default' ]]; then travis_retry composer require --dev --update-with-dependencies --prefer-dist --no-interaction --no-suggest laravel/laravel "$laravel"; fi
+  - composer info | grep -e laravel/laravel -e phpunit/phpunit -e phpstan/phpstan
```
Comment (tarampampam) [inline .travis.yml]: Not `laravel/laravel`. `illuminate/support`

File: .travis.yml [hunk 5]
```diff
@@ -32,26 +35,41 @@ after_success:
...
+    - php: 7.3
+      env: laravel=5.5
+    - php: 7.3
+      env: laravel=5.6
+    - php: 7.3
+      env: laravel=5.7
+    - php: 7.3
+      env: laravel=5.8
+    - php: 7.3
+      env: coverage=true laravel=6.*
```
Comment (tarampampam) [inline .travis.yml]: Move prev. varsion back. *Just* add matrix cases:
- With `php: 7.2 + env: illuminzte=6.*`
- With `php: 7.3 + env: illuminzte=6.*`

File: .travis.yml (hunk 1 above)
Comment (korobovn) [reply .travis.yml]: yes

File: .travis.yml (hunk 2 above)
Comment (korobovn) [reply .travis.yml]: ok

File: .travis.yml (hunk 3 above)
Comment (korobovn) [reply .travis.yml]: ok

File: .travis.yml (hunk 4 above)
Comment (korobovn) [reply .travis.yml]: ok

File: .travis.yml (hunk 5 above)
Comment (korobovn) [reply .travis.yml]: ok

**Pull request merged**